controllers/secretproviderclasspodstatus_controller_test.go

```diff
@@ -172,7 +172,7 @@ func TestPatchSecretWithOwnerRef(t *testing.T) {
 	client := fake.NewFakeClientWithScheme(scheme, initObjects...)
 	reconciler := newReconciler(client, scheme, "node1")
 
-	err = reconciler.patchSecretWithOwnerRef(context.TODO(), "my-secret", "default", ref)
+	err = reconciler.patchSecretWithOwnerRef(context.TODO(), "my-secret", "default", ref, ref)
```

Comment:
nit: a comment here explaining why `ref` is passed twice (or a separate test case for the double ref)

Reply:
added a comment